### PR TITLE
Update table of contents

### PR DESCRIPTION
**Pull request by muhammadhafiz18**
- Updated links in the table of contents for Semantic Versioning and CRON, including new icons.

- Removed outdated content from the CRON article.

### CODE REVIEW

File: tooling/index.md [hunk 1]
```diff
@@ -7,5 +7,7 @@ Here you will find a collection of useful tools and resources for your developme
         <a href="vscode-install.md" class="list-group-item list-group-item-action">VS Code o'rnatish ✨</a>
         <a href="vscode-setup.md" class="list-group-item list-group-item-action">VS Code sozlash 🪛</a>
         <a href="multiple-git-remotes.md" class="list-group-item list-group-item-action">Bir nechta <strong>git remote</strong> bilan ishlash 🖇️</a>
+        <a href="semantic-versioning.md" class="list-group-item list-group-item-action">VS Code sozlash 🪛</a>
+        <a href="cron.md" class="list-group-item list-group-item-action">VS Code sozlash 🪛</a>
```
Comment (wahidustoz) [inline tooling/index.md]:
```suggestion
        <a href="semantic-versioning.md" class="list-group-item list-group-item-action">Versiyalarni raqamlash 🔢</a>
        <a href="cron.md" class="list-group-item list-group-item-action">Cron bilan vaqtni boshqarish ⏰</a>
```

**Review by wahidustoz — CHANGES_REQUESTED** [review]

File: tooling/CRON.md [hunk 2]
```diff
@@ -107,11 +107,8 @@ https://t.me/MuazzinUz_bot da esa dastur har daqiqada ishlaydi. Namoz vaqtini ol
...
 
----
-
-### Shunday qilib, CRON bu dasturlarni ma'lum vaqtda ishlatish uchun tool hisoblanar ekan. Dasturni yilning istalgan oyining istalgan kuning istalgan soatida ishga tushadigan qilib qo'ysa bo'ladi.
+--- 
 
----
 Xurmat bilan,
 
 Muhammd Khodjaev
```
Comment (wahidustoz) [inline tooling/CRON.md]: Ism familiya kerak emas

File: toc.yml
Comment (wahidustoz) [inline toc.yml:1]: `fluent-validation.md` ni toc.yml ga qo'shmapsiz.

**Review by wahidustoz — CHANGES_REQUESTED** [review]

File: tooling/CRON.md (hunk 2 above)
Comment (muhammadhafiz18) [reply tooling/CRON.md]: aha

**Review by wahidustoz — APPROVED** [review]

**Pull request merged**